eq-author/src/App/page/Routing/DestinationSelector/RoutingDestinationContentPicker.test.js

```diff
@@ -25,70 +80,132 @@ describe("RoutingDestinationContentPicker", () => {
       },
       loading: false,
     };
-
-    wrapper = render(props);
   });
 
   it("should render", () => {
-    expect(wrapper).toMatchSnapshot();
+    const { getByText, getByTestId } = render(
+      <UnwrappedRoutingDestinationContentPicker {...props} />
+    );
+    const openButton = getByTestId("content-picker-select");
```

Comment:
Just wondering if there is a way to create this variable as part of props so it only needs creating once?
I do the same too, so just a general question really!

Reply:
I could do something like this ?
```js
    let openButton, destinationPicker;
    beforeEach(() => {
      destinationPicker = render(
        <UnwrappedRoutingDestinationContentPicker {...props} />
      );
      openButton = destinationPicker.getByTestId("content-picker-select");
    });

    it("should only display questions after the current question in the section", () => {
      const { getAllByText, queryByText } = destinationPicker;
      fireEvent.click(openButton);
      expect(queryByText("s1q1")).toBeFalsy();

      getAllByText("s1q3");
      getAllByText("s1q4");
    });

    it("should only display first question in other sections", () => {
      const { getAllByText, queryByText, getByText } = destinationPicker;
      fireEvent.click(openButton);

      const section2Button = getByText("section2");
      fireEvent.click(section2Button);
      getAllByText("s2q1");
      expect(queryByText("s2q2")).toBeFalsy();

      const section3Button = getByText("section3");
      fireEvent.click(section3Button);
      getAllByText("s3q1");
      expect(queryByText("s3q2")).toBeFalsy();
    });
```

Reply:
I pushed that up now :) Shaved off 2 lines 😉